<!-- -->
<launch>
  <!-- <param name="/use_sim_time" value="true" /> -->
  <arg name="role_name" default="ego_vehicle" />

  <node pkg="controller" type="controller" name="controller" output="log">
    <param name="role_name" type="string" value="$(arg role_name)" />
    
    <rosparam command="load" file="$(find controller)/config/controller_config.yaml"/>

  </node>
</launch>
